### ISSUE TITLE
It gets stuck after creating apikey successfully

### ISSUE THREAD

**Issue by j-fuentes** [digitalocean/terraform-provider-sendgrid#2]
I am using this to create an apikey:

```
terraform {
  required_version = "= 0.12.24"
}

provider "sendgrid" {
  api_key = "<my-apikey>"
}

resource "sendgrid_api_key" "api-key" {
  name = "test-apikey-example"
  scopes = [ "mail.send" ]
  destination = "./apikey"
}

```

**What happened:**

The apikey is created successfully, but the provider gets stuck and finished with error:

```
$ terraform apply

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # sendgrid_api_key.api-key will be created
  + resource "sendgrid_api_key" "api-key" {
      + destination = "./apikey"
      + id          = (known after apply)
      + name        = "test-apikey-example"
      + scopes      = [
          + "mail.send",
        ]
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

sendgrid_api_key.api-key: Creating...
sendgrid_api_key.api-key: Still creating... [10s elapsed]
sendgrid_api_key.api-key: Still creating... [20s elapsed]
sendgrid_api_key.api-key: Still creating... [30s elapsed]
sendgrid_api_key.api-key: Still creating... [40s elapsed]
sendgrid_api_key.api-key: Still creating... [50s elapsed]
sendgrid_api_key.api-key: Still creating... [1m0s elapsed]
sendgrid_api_key.api-key: Still creating... [1m10s elapsed]
sendgrid_api_key.api-key: Still creating... [1m20s elapsed]
sendgrid_api_key.api-key: Still creating... [1m30s elapsed]
sendgrid_api_key.api-key: Still creating... [1m40s elapsed]
sendgrid_api_key.api-key: Still creating... [1m50s elapsed]
sendgrid_api_key.api-key: Still creating... [2m0s elapsed]
sendgrid_api_key.api-key: Still creating... [2m10s elapsed]
sendgrid_api_key.api-key: Still creating... [2m20s elapsed]
sendgrid_api_key.api-key: Still creating... [2m30s elapsed]
sendgrid_api_key.api-key: Still creating... [2m40s elapsed]
sendgrid_api_key.api-key: Still creating... [2m50s elapsed]
sendgrid_api_key.api-key: Still creating... [3m0s elapsed]
sendgrid_api_key.api-key: Still creating... [3m10s elapsed]
sendgrid_api_key.api-key: Still creating... [3m20s elapsed]
sendgrid_api_key.api-key: Still creating... [3m30s elapsed]

Error: Error waiting for key test-apikey-example (id: RSeN-bqDS5SO2tmmDtEBEw) to be created: couldn't find resource (21 retries)

  on terraform.tf line 9, in resource "sendgrid_api_key" "api-key":
   9: resource "sendgrid_api_key" "api-key" {
```

For what is worth, the API token was written to the destination file.

**What I expected to happen:**

I expect it to create the apikey, write the credentials to file and finish.

**Comment by j-fuentes** [comment]
I have solved this problem in #3. Check the PR description to see the details of the problem.

**Comment by sssd-dev** [comment]
I am experiencing the same issue. Can we please go ahead and merge in master?

**Comment by jrm780** [comment]
This should be resolved by #3 and is built into release [v0.0.2](https://github.com/digitalocean/terraform-provider-sendgrid/releases/tag/v0.0.2).

**Comment by j-fuentes** [comment]
thanks!
